File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/Model/DTO/ReponseDTO/TemplateFieldResponseDTO.java
package com.example.APIRollTheDice.Model.DTO.ReponseDTO;


import com.example.APIRollTheDice.Model.DTO.GameDTO.TemplateDTO.OptionListDTO;
import com.example.APIRollTheDice.Model.DTO.GameDTO.TemplateDTO.TemplateFieldDTO;



public class TemplateFieldResponseDTO {
    private TemplateFieldDTO templateField;
    private OptionListDTO optionList;

    public TemplateFieldResponseDTO(TemplateFieldDTO templateField, OptionListDTO optionList) {
        this.templateField = templateField;
        this.optionList = optionList;
    }

    public TemplateFieldResponseDTO() {
    }

    public TemplateFieldDTO getTemplateField() {
        return templateField;
    }

    public void setTemplateField(TemplateFieldDTO templateField) {
        this.templateField = templateField;
    }

    public OptionListDTO getOptionList() {
        return optionList;
    }

    public void setOptionList(OptionListDTO optionList) {
        this.optionList = optionList;
    }
}

File: RollTheDice/Assets/_Project/API/Model/DTO/ReponseDTO/TemplateFieldResponseDTO.cs
using RollTheDice.API.DTO.GameDTO.TemplateDTO;

namespace RollTheDice.API.DTO.ReponseDTO
{
    public class TemplateFieldResponseDTO
    {
        public TemplateFieldDTO TemplateField { get; set; }
        public OptionListDTO OptionList { get; set; }

        public TemplateFieldResponseDTO(){}
        public TemplateFieldResponseDTO(TemplateFieldDTO templateFieldDTO, OptionListDTO optionList)
        {
            TemplateField = templateFieldDTO;
            OptionList = optionList;
        }
    }
}